### ISSUE TITLE
Plugin support for notifications?

### ISSUE THREAD

**Issue by strarsis** [wordpress-mobile/WordPress-Android#10447]
Can a plugin send notifications, like when a form is submitted by Caldera Forms?
There is a separate app for WooCommerce, but for something like a form plugin it would make more sense to notify using the WordPress app.

**Comment by designsimply** [comment]
No, not right now. This is a thing that might make that possible down the road: https://make.wordpress.org/core/2019/08/05/feature-project-proposal-wp-notify/
<sup>(internal reference: C02QANACA/p1568924463002100-slack-android-dev)</sup>

Let's re-frame this from a question to a request to implement generic notification support once the Core WP Notify project ships. 🙂

**Comment by develric** [comment]
I'm placing a comment here to remove the current stale state. I think this request is still worth to be kept alive since the mentioned wp-notify project is progressing on the backend and frontend web side of things, so worth to revisit this in a bit to evaluate mobile side implications eventually.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
